src/Tickets/Commerce/Order_Modifiers/Admin/Order_Modifier_Fee_Metabox.php

```diff
@@ -149,6 +150,11 @@ public function enqueue_order_modifiers_fee_scripts() {
 	 * @return void
 	 */
 	public function add_fee_section( int $post_id, ?int $ticket_id ): void {
+		$provider = Tribe__Tickets__Tickets::get_event_ticket_provider( $post_id );
+
+		if ( 'TEC\Tickets\Commerce\Module' !== $provider ) {
```

Comment:
Two minor adjustments to how we import these other classes:

1. `Tribe__Tickets__Tickets` can be aliased to just `Tickets`
2. We should import the `TEC\Tickets\Commerce\Module` class. An alias here is optional, but it could be aliased to `Commerce_Module`. Then we use the `::class` constant to reference it.

```suggestion
		$provider = Tickets::get_event_ticket_provider( $post_id );

		if ( Module:class !== $provider ) {
```